index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Leading Design Festival</title>
+</head>
+<body>
+  <header>
+  <button type="button" class="hamburger">
+    <span></span>
+    <span></span>
+    <span></span>
+  </button>
+  <div class="dNav">
+    <a class = "brand" href="index.html"><img src="./Logo/ld.png" alt=""> Leading Design Festival</a>
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about.html">About</a></li>
+      <li><a href="">Program</a></li>
+      <li><a href="">Join</a></li>
+      <li><a href="">Sponsor</a></li>
+    </ul>
+    <a class="Campaign" href="#">LD Campaign</a>
+  </div>
```

Comment:
- [ ]  The HTML uses semantic tags, the header use `header` tags, sections use `section` articles use `article`, the nav and footer use `nav` and `footer`..., Please change the `<div class="dNav">` tags into `<nav class="dNav">` 👍🏻 It could be something like this :point_down:

```
   <header>
      <nav class="dNav">
         <ul>
           <li><a href="index.html">Home</a></li>
           <li><a href="about.html">About</a></li>
                   ...
          </lu>
      </nav>
  </header>
```
- I know you can do it 😄 💪🏻